changelog/1200.bugfix.rst

```diff
@@ -0,0 +1 @@
+Do not use :class:`.abc.PrivateChannel` in an ``isinstance`` check as it structurally matches :class:`Thread`.
```

Comment:
```suggestion
|commands| Fix usage of :attr:`~ext.commands.BucketType.role`\-type cooldowns in threads, which incorrectly operated on a per-channel basis instead.
```
The inner workings are pretty opaque, for a user-facing changelog something like this might be better.